data/hulls/ship_data.csv

```diff
@@ -7,50 +7,50 @@ Arbitrator,diableavionics_arbitrator,,Diable Avionics,diableavionics_booster,,30
 Blizzaia,diableavionics_blizzaia,,Diable Avionics,flarelauncher_active,,1600,150,500,,100,0,,155,150,75,50,75,70,NONE,,,,,,,6,6,,,,,,,,,,,,,,,,,,,,11
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Valiant,diableavionics_valiant,,Diable Avionics,diableavionics_transit,,800,75,1200,,200,0,,160,200,200,90,180,45,OMNI,,120,0.5,1,,,3,3,,,,,,,,,,,,,,,,,,,,9
-Raven,diableavionics_raven,,Diable Avionics,diableavionics_flicker,,2000,225,1250,,250,0,,120,120,80,50,50,75,NONE,,,,,,,6,6,,,,,,,,,,,,,,,,,,,,10
-Zephyr,diableavionics_zephyr,,Diable Avionics,diableavionics_splitter,,1500,175,750,,200,0,,165,240,150,75,150,65,NONE,,,,,,,5,5,,,,,,,,,,,,,,,,,,,,9
+Raven,diableavionics_raven,,Diable Avionics,diableavionics_flicker,,3000,300,1250,,250,0,,120,120,80,50,50,75,NONE,,,,,,,6,6,,,,,,,,,,,,,,,,,,,,10
+Zephyr,diableavionics_zephyr,,Diable Avionics,diableavionics_splitter,,1400,100,2000,,300,0,,180,240,150,75,150,65,OMNI,,240,0.5,1,,,5,5,,,,,,,,,,,,,,,,,,,,9
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Valiant,diableavionics_valiantTrainer,Trainer,Diable Avionics,diableavionics_transitTrainer,,800,75,1200,,200,0,,160,200,200,90,180,45,OMNI,,120,0.5,1,,,3,3,,,,,,,,,,,,,HIDE_IN_CODEX,restricted,,,,,,9
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Draft,diableavionics_draft,Frigate,Diable Avionics,diableavionics_draftrepairs,4,2250,300,1500,,190,50,,120,90,60,60,60,275,NONE,,0,0,0,,,25,35,50,25,1,,10,8000,7,15,300,0.2,4,4,SHIP_WITH_MODULES,diable_ship_bp,,0.25,2,2,,13
-Arcus,diableavionics_draftarmormodule,,,,,4000,400,0,,0,0,,400,400,400,200,400,125,NONE,,,0,0,,,,,,,,,,,,,,,0,0,HIDE_IN_CODEX,module_hull_bar_only,,,,,,4
-Vapor,diableavionics_vapor,Frigate,Diable Avionics,diableavionics_evasion,6,1500,200,2400,,225,50,,175,130,110,100,80,150,OMNI,,90,0.5,1,,,20,30,35,30,1,,10,15000,10,20,210,0.2,6,6,,"diable_ship_bp,rare_bp",,0.5,2,2,,14
-Sleet,diableavionics_sleet,Frigate,Diable Avionics,diableavionics_capacitors,5,2000,250,3000,,150,50,,140,120,100,80,80,200,OMNI,,120,0.4,1,,,30,40,80,35,1,,10,13000,8,16,240,0.2,5,5,,"diable_ship_bp,rare_bp",,0.5,2,2,,14
+Draft,diableavionics_draft,Frigate,Diable Avionics,diableavionics_draftrepairs,4,3000,300,1500,,190,50,,120,90,60,60,60,275,NONE,,0,0,0,,,25,35,50,25,1,,10,8000,7,15,300,0.2,4,4,SHIP_WITH_MODULES,diable_ship_bp,,0.25,2,2,,13
+Arcus,diableavionics_draftarmormodule,,,,,4000,600,0,,0,0,,400,400,400,200,400,125,NONE,,,0,0,,,,,,,,,,,,,,,0,0,HIDE_IN_CODEX,module_hull_bar_only,,,,,,4
+Vapor,diableavionics_vapor,Frigate,Diable Avionics,diableavionics_evasion,6,1500,200,3000,,225,50,,175,130,110,100,80,150,OMNI,,120,0.5,1,,,20,30,35,30,1,,10,15000,10,20,210,0.2,6,6,,"diable_ship_bp,rare_bp",,0.5,2,2,,14
+Sleet,diableavionics_sleet,Frigate,Diable Avionics,diableavionics_capacitors,5,2000,250,3000,,150,50,,140,120,100,80,80,200,OMNI,,180,0.4,0.8,,,30,40,80,35,1,,10,13000,8,16,240,0.2,5,5,,"diable_ship_bp,rare_bp",,0.5,2,2,,14
```

Comment:
Draft hull increased to 3000 (from 2250), module armor increased to 600 (from 400). It also has Rugged Construction now, which is a change that I made.
Vapor flux capacity increased to 3000 (from 2400), shield arc increased to 120 (from 90).
Sleet shield arc increased to 180 (from 120), shield efficiency increased to 0.8 (from 1).